ext/standard/math.c

```diff
@@ -170,56 +122,30 @@ static inline double php_round_helper(double value, int mode) {
  * mode. For the specifics of the algorithm, see http://wiki.php.net/rfc/rounding
  */
 PHPAPI double _php_math_round(double value, int places, int mode) {
-	double f1, f2;
+	double f1;
```

Comment:
Silly nit, but maybe a more descriptive variable name for this would be helpful?

Maybe like `exponent` ?

Reply:
It's true that I have no idea what `f1` means. It's already late today, so I'll change it tomorrow.